aws/provider.go

```diff
@@ -412,8 +412,8 @@ func Provider() terraform.ResourceProvider {
 			"aws_elastic_beanstalk_environment":                resourceAwsElasticBeanstalkEnvironment(),
 			"aws_elasticsearch_domain":                         resourceAwsElasticSearchDomain(),
 			"aws_elasticsearch_domain_policy":                  resourceAwsElasticSearchDomainPolicy(),
-			"aws_elastictranscoder_pipeline":                   resourceAwsElasticTranscoderPipeline(),
```

Comment:
Renaming a resource without creating an alias resource referencing the existing resource is a breaking change.